### PR TITLE
Add GCP to sources

### PR DESCRIPTION
**Pull request by rvsia**
TODO:
- [x] finalize API https://github.com/RedHatInsights/sources-api/pull/299
- [x] tests
- [x] fix icon position in selector
- [x] **remove mocked API data**

image
image
image
image
image
image
image

### CODE REVIEW

**Comment by beccaglass** [comment]
@rvsia is 'BiqQuery' a typo?

**Comment by beccaglass** [comment]
@rvsia (almost) finalized copy changes: https://marvelapp.com/prototype/1360h9j9/section/1188068

I say almost because Doug is still on PTO so haven't run the final flow by him yet to make sure the process is still correct.

On a more general note, I'm going to go through all the iterations of the wizard and do a SPUR - point out the differences / make sure everything is consistent where it needs to be.

File: packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    }, { b })}
+                </TextListItem>
+                <TextListItem>
+                    { intl.formatMessage({
+                        id: 'cost.gcp.access.secondStep',
+                        defaultMessage: 'Paste the following value into the <b>Add members</b> field:'
+                    }, { b })}
+                </TextListItem>
+                <ClipboardCopy className="pf-u-m-sm  pf-u-ml-0" isReadOnly>
+                    billing-export@red-hat-cost-management.iam.gserviceaccount.com
```
Comment (dccurtis) [inline packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: Is it possible for the service account email to be seeded via an environment variable?

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: Maybe, not sure. I have never done it. (Passing env variables to dependencies.)

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: However, if you need to change this email according to env, we can do it here.

Comment (dccurtis) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: We'll have a different service account email for production.  Do you have any suggestions on how best to handle this?

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: If it isn’t going to be changed a lot we can hardcoded it here (if prod use this email). Or fetch it from an api. (As we are already fetching data for SubWatch from their api.)

Comment (dccurtis) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:92]: ok got it.  I think we might go the API fetching route.  We basically want to have one service account for development (CI/QA) and one for production and staging.

Comment (dccurtis) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:151]: @rvsia if we are going the hard-coded route initially can you update this to be:
`test-billing-service-account@cloud-billing-292519.iam.gserviceaccount.com`
Thanks!

Comment (rvsia) [reply packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js:151]: Ok, will change it next week!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**